web_widget_open_tab/static/src/less/open_tab.less

```diff
@@ -0,0 +1,10 @@
+.open_tab_widget {
```

Comment:
can't you toggle `.invisible/.visible` class on the element via JS instead of loading another resource?
Not a blocker but we reduce the size of the assets (and save 1 request in debug=assets mode).

Reply:
True. Also, tooltip is doing it directly, so it has no sense.